package.json

```diff
@@ -7,6 +7,7 @@
     "test": "tests"
   },
   "dependencies": {
+    "@types/fs-extra": "^2.1.0",
     "@types/webassembly-js-api.d.ts": "https://github.com/winksaville/webassembly-js-api.d.ts",
```

Comment:
You can use our "official" version in DefinitelyTyped which went in yesterday and put it in devDependencies":
```
yarn remove @types/webassembly-js-api.d.ts
yarn add @types/webassembly-js-api -D
```